.github/workflows/main.yml

```diff
@@ -12,19 +12,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.7, '3.0', 3.1, 3.2]
-        rails: ['6.0', '6.1', '7.0', '7.1']
+        ruby: [2.7, '3.0', 3.1, 3.2, 3.3]
+        rails: ['6.0', '6.1', '7.0', '7.1', '7.2']
         include:
           - ruby: 2.6
             rails: '5.2'
+        exclude:
+          # rails 7.2: support ruby 3.1+
+          - ruby: 2.7
+            rails: '7.2'
+          - ruby: 3.0
+            rails: '7.2'
 
     name: Ruby ${{ matrix.ruby }} - Rails ${{ matrix.rails }}
     env:
       BUNDLE_GEMFILE: ${{ github.workspace }}/gemfiles/rails_${{ matrix.rails }}.gemfile
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Updated `actions/checkout@v4` to fix bellow warning.
> The following actions use a deprecated Node.js version and will be forced to run on node20: actions/checkout@v3. For more info: https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/

Reply:
Awesome, thanks!